source/includes/_pis.md

```diff
@@ -55,6 +55,26 @@ We support account identification using `UK.OBIE.SortCodeAccountNumber`. We don'
 
 You can only make payments in `GBP`. We don't support other currencies.
 
+### Domestic Payments Limits
+
+There are default limits for daily outbound payments:
+
+| Account type                       | Default limit |
+| -----------------------------------| ------------- |
+| Personal                           | £10000        |
+| Joint                              | £10000        |
+| Business (sole trader)             | £25000        |
+| Business (private limited company) | £50000        |
+
+<aside class="notice">
+Please note that limits for business account are different depending on business company type.  See <a href="#business-account-company-types">Account Information Services API - Business company types</a> for details.
+</aside>
+
+<aside class="notice">
+Please note that those are <b>default</b> limits applied to payments and can be increased if asked by customer.
```

Comment:
nit: "Please note that **these** are default limits applied to payments and can be increased if asked by **the** customer.

Reply:
thanks!